### PR TITLE
fix: adds correct cursor pointer for the play/pause controlbar button. 

### PR DESCRIPTION
**Pull request by gjanblaszczyk**
## Description
A proposed fix for videojs/video.js#5462

## Requirements Checklist
- [X] Feature implemented / Bug fixed
- [ ] If necessary, more likely in a feature request than a bug fix
  - [X] Change has been verified in an actual browser (Chome, Firefox, IE)
  - [ ] Unit Tests updated or fixed
  - [ ] Docs/guides updated
  - [ ] Example created ([starter template on JSBin](http://jsbin.com/axedog/edit?html,output))
- [ ] Reviewed by Two Core Contributors

### CODE REVIEW

File: src/css/components/_play-pause.scss [hunk 1]
```diff
@@ -1,3 +1,6 @@
+.video-js .vjs-play-control {
+  cursor: pointer;
+}
 .video-js .vjs-play-control .vjs-icon-placeholder {
   cursor: pointer;
```
Comment (gkatsev) [inline src/css/components/_play-pause.scss]: this line is probably no longer necessary.

Comment (gjanblaszczyk) [reply src/css/components/_play-pause.scss]: Ah, you are right. Fixed now.

**Comment by misteroneill** [comment]
Thanks!

**Comment by jmcastor** [comment]
https://youtu.be/OOHstPceQW0

**Pull request merged**